src/resources/parser/glb-parser.js

```diff
@@ -958,27 +955,20 @@ const createMaterial = function (gltfMaterial, textures, disableFlipV) {
             }
         }
 
-        let scale = uvONE;
-        let offset = uvZERO;
+        const textureTransform = source.extensions?.KHR_texture_transform;
+        if (textureTransform) {
+            const offset = textureTransform.offset || [0, 0];
+            const scale = textureTransform.scale || [1, 1];
+            const rotation = textureTransform.rotation ? (-textureTransform.rotation * math.RAD_TO_DEG) : 0;
 
-        const extensions = source.extensions;
-        if (extensions) {
-            const textureTransformData = extensions.KHR_texture_transform;
-            if (textureTransformData) {
-                if (textureTransformData.scale) {
-                    scale = textureTransformData.scale;
-                }
-                if (textureTransformData.offset) {
-                    offset = textureTransformData.offset;
-                }
-            }
-        }
+            const tilingVec = new Vec2(scale[0], scale[1]);
+            const offsetVec = new Vec2(offset[0], 1.0 - scale[1] - offset[1]);
```

Comment:
I would combine these together. There's no need to create arrays to later convert them to Vec2. Go to Vec2 directly.

Reply:
I wouldn't get too obsessed with extreme optimisation of loading code that is called maybe twice during the loading of a glTF file. Being code concise is much more important in this case.

Please offer a code suggestion though, perhaps I'm missing something.

Reply:
Agreed. I offered a code before for this section. Something like:
`const tilingVec = textureTransform.offset ? new Vec2(textureTransform.offset[0], textureTransform.offset[1]) : Vec2.ZERO;`

Reply:
And for offset?

Reply:
hmm, same thing?
`const offsetVec = textureTransform.offset ? new Vec2(textureTransform.offset.x, 1.0 - textureTransform.offset[1] - tilingVec.y) : Vec2.ONE;`

Reply:
I'm using `Vec2.ZERO` and `Vec2.ONE` instead of inline arrays, which I think gives us both what we want.